assets/src/edit-story/components/fontPicker/pickerContainer.js

```diff
@@ -117,40 +172,40 @@ const Item = styled.div.attrs(({ fontFamily }) => ({
   },
 }))`
   letter-spacing: ${({ theme }) => theme.fonts.label.letterSpacing};
-  padding: 8px 12px 8px 26px;
+  padding: 8px 16px;
   margin: 0;
   white-space: nowrap;
   overflow: hidden;
   color: ${({ theme }) => theme.colors.fg.white};
-  font-size: ${({ theme }) => theme.fonts.label.size};
-  line-height: ${({ theme }) => theme.fonts.label.lineHeight};
+  font-size: ${({ theme }) => theme.fonts.body2.size};
   font-weight: ${({ theme }) => theme.fonts.label.weight};
+  line-height: 1;
   position: relative;
   cursor: pointer;
+  background-clip: padding-box;
 
   &:hover,
   &:focus {
-    background-color: ${({ theme }) => theme.colors.bg.v15};
+    background-color: ${({ theme }) => rgba(theme.colors.bg.white, 0.1)};
     outline: none;
   }
 `;
 
 const Selected = styled(Checkmark)`
-  position: absolute;
-  left: 0;
-  top: 3px;
-  width: 28px;
-  height: 28px;
+  display: inline-block;
+  width: 8px;
+  height: auto;
+  margin-right: 8px;
 `;
 
-const NoResult = styled.span`
-  letter-spacing: ${({ theme }) => theme.fonts.label.letterSpacing};
-  padding: 15px 12px 15px 26px;
+const NoResult = styled.div`
+  width: 100%;
+  padding: 13px 11px;
   margin: 0;
-  font-style: italic;
-  color: ${({ theme }) => theme.colors.fg.white};
-  font-size: ${({ theme }) => theme.fonts.body1.size};
-  line-height: ${({ theme }) => theme.fonts.body1.lineHeight};
+  text-align: center;
+  color: ${({ theme }) => rgba(theme.colors.fg.white, 0.75)};
+  font-size: ${({ theme }) => theme.fonts.tab.size};
+  line-height: 14px;
 `;
 
 function FontPickerContainer({ value, onSelect, onClose, isOpen }) {
```

Comment:
This file is very long now (it also was before, but it's even worse now). Can we split it up on some smaller logical sub-units? At least get this file below 200 lines as a start, that would be awesome!

Reply:
Awesome rewrite here, excellent job in creating an overview and an easy understanding of the components and their concerns.